### PR TITLE
Deprecate this repo

### PR DESCRIPTION
**Pull request by csibbitt**
Active work has moved here: https://github.com/redhat-service-assurance/service-assurance-operator/

Also, see this PR: https://github.com/redhat-service-assurance/service-assurance-operator/pull/34

More detail on Trello 898

### CODE REVIEW

**Comment by csibbitt** [comment]
[Move testing (and quickstart) artifacts from telemetry-framework to SAO repo](https://trello.com/c/ZpAvTdgM/898-move-testing-and-quickstart-artifacts-from-telemetry-framework-to-sao-repo)

File: README.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-of various platform components (not applications) in order to provide a
-centralized view of multiple platform deployments.
+Active work has moved here: https://github.com/redhat-service-assurance/service-assurance-operator/
 
-## Development
-
-The quickest way to start up Service Assurance Framework is to run the
-`quickstart.sh` script located in the `deploy/` directory after starting up a
-[Code Ready Containers](https://github.com/code-ready/crc) environment.
+The contents of this repository remain in orer to preserve WIP patches (please delete your branches when you're done) and supporting existing Tech Preview installations.
```
Comment (leifmadsen) [inline README.md]: "in order"

File: README.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 ## Tech Preview
+
 See the [official
 documentation](https://redhat-service-assurance.github.io/saf-documentation)
 for more information about installing for production-style use cases on OCP3.
 
-Please use the `saf-ocp3` branch for all such installations. 
+Please use [the legacy saf-ocp3 branch](https://github.com/redhat-service-assurance/telemetry-framework/tree/saf-ocp3) for all such installations.
+
+As of the SAF GA release, this repository is deprecated.
```
Comment (leifmadsen) [inline README.md]: I'd call out GA == general availability since GA might be a Red Hat ism

**Review by leifmadsen — APPROVED** [review]
Approved pending 2 minor changes.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,15 @@
-# telemetry-framework  [![Build Status](https://travis-ci.org/redhat-service-assurance/telemetry-framework.svg?branch=master)](https://travis-ci.org/redhat-service-assurance/telemetry-framework)
+# DEPRECATED REPOSITORY telemetry-framework
```
Comment (leifmadsen) [inline README.md]: :+1:

**Comment by mrunge** [comment]
I am fine with deprecating this, once the minors in the other PR are addressed.

File: README.md (hunk 2 above)
Comment (csibbitt) [reply README.md]: Done, but FYI https://www.acronymfinder.com/General-Availability-(GA).html

**Pull request merged**